home-screen-widget/profiles/README.md

```diff
@@ -0,0 +1,43 @@
+# Home Screen Example Widget
+
+This is a guided example on how to develop a home screen widget.
+In this example we provide a working widget that uses the Beekeeper Javascript SDK to fetch 50 user profiles and display
+them in a grid layout.
+
+![](https://github.com/beekpr/examples/blob/FUL-21524/home-screen-widget/profiles/photos/profiles_widget.png)
```

Comment:
This should be changed to a relative URL

Reply:
A cool I didn't know you could do that. thanks